src/vos/tests/vts_mvcc.c

```diff
@@ -33,6 +33,19 @@
 #include "vts_io.h"
 #include "vts_array.h"
 
+struct tx_helper {
+	/** Current transaction handle */
+	struct dtx_handle	*th_dth;
+	/** Number of total ops in current tx */
+	uint32_t		 th_nr_ops;
+	/** Number of write ops in current tx */
+	uint32_t		 th_nr_mods;
```

Comment:
Why not use dth::dth_modification_cnt ?

Reply:
The txh is used to create the dth so it's a chicken and egg problem.